.drone.yml

```diff
@@ -95,7 +95,7 @@ steps:
     dockerfile: package/Dockerfile.webhook
     password:
       from_secret: docker_password
-    repo: "rancher/harvester-webhook"
+    repo: "joshmeranda/harvester-webhook"
```

Comment:
@joshmeranda I guess we can keep all repos with existing values. And don't worry that the PR needs to be tested with another installer PR, reviewers can handle this.

Reply:
And similar changes in other files.

Reply:
reversed changes to those files